lib/Benchmark/Remote/Payload.php

```diff
@@ -70,6 +70,10 @@ class Payload
      */
     public function __construct($template, array $tokens = [], $phpBinary = PHP_BINARY, Process $process = null)
     {
+        if (PHP_OS === 'WINNT') {
+            $phpBinary = sprintf('"%s"', $phpBinary);
```

Comment:
I think it would be better to do this?

```php
$this->phpBinary = escapeshellarg($phpBinary)
```

?

Reply:
I first tried escapeshellcmd and escapeshellarg, unfortunately both failed so i went with this manual solution.

Reply:
Sorry, i just double-checked my former statement and `escapeshellarg` works like a charm. I cleaned that up.